tasks/main.yml

```diff
@@ -31,7 +31,7 @@
 
 - name: check current version
   command: >
-    {{ helm_install_dir }}/helm version --client --template {{ "'{{ .Client.SemVer }}'" }}
+    {{ helm_install_dir }}/helm version --client --template {{ "'{{ .Version }}'" }}
```

Comment:
This appears to be a breaking change in Helm 3. If you change it to the following it'll work in Helm 2 and Helm 3:
```
"'{{ if .Version }}{{ .Version }}{{ else }}{{ .Client.SemVer }}{{ end }}'"
```